inflammation/models.py

```diff
@@ -32,3 +32,13 @@ def daily_min(data):
     """Calculate the daily min of a 2d inflammation data array."""
     return np.min(data, axis=0)
 
+
+def s_dev(data):
```

Comment:
```suggestion
def standard_deviation(data):
```
The function could replaced by ```np.std()```, adds to the readability of the code! If you think you need the whole function for some reason, consider changing its name to something more understandable! (for e.g. ```standard_deviation()```